### DISCUSSION TITLE
자바스크립트에서 실행 컨텍스트가 어떻게 코드 실행에 영향을 미치는지 설명해 주실 수 있나요?

### DISCUSSION THREAD

**Discussion by lledellebell** [diving-javascript/you-dont-know-js-yet#21] [실행 컨텍스트(Execution Context)]
- #18

**Comment by wogus216** [comment]
자바스크립트에서 실행 컨텍스트는 코드가 실행되는 환경을 정의하고, 코드 실행 중에 발생하는 다양한 정보를 관리하는 매우 중요한 개념입니다. 이는 자바스크립트 엔진이 코드를 처리하는 방식을 이해하는 데 핵심적인 역할을 합니다. 실행 컨텍스트는 코드에 대한 정보를 캡슐화하고, 각 코드 블록이나 함수가 실행될 때 필요한 모든 정보를 제공합니다
